src/FFmpegReader.cpp

```diff
@@ -706,9 +706,12 @@ void FFmpegReader::UpdateVideoInfo() {
 	info.vcodec = pCodecCtx->codec->name;
 	info.video_bit_rate = (pFormatCtx->bit_rate / 8);
 
-	// set frames per second (fps)
-	info.fps.num = pStream->avg_frame_rate.num;
-	info.fps.den = pStream->avg_frame_rate.den;
+	// Frame rate from the container and codec
+	AVRational framerate = av_guess_frame_rate(pFormatCtx, pStream , NULL);
```

Comment:
```suggestion
	AVRational framerate = av_guess_frame_rate(pFormatCtx, pStream, NULL);
```
Remove the spurious space character, please.